build.sh

```diff
@@ -247,7 +247,12 @@ if [ $BUILD_MAC == 1 ]; then
 	# Use our own updater, because Mozilla's requires updates signed by Mozilla
 	cd "$CONTENTSDIR/MacOS"
 	tar -xjf "$CALLDIR/mac/updater.tar.bz2"
-	
+
+	# Copy PDF tools and data
+	cp "$CALLDIR/pdftools/pdftotext-mac" "$CONTENTSDIR/MacOS/pdftotext"
+	cp "$CALLDIR/pdftools/pdfinfo-mac" "$CONTENTSDIR/MacOS/pdfinfo"
+	cp -r "$CALLDIR/pdftools/poppler-data" "$CONTENTSDIR/Resources/"
```

Comment:
Doesn't really matter (and it's wrong a few more places in this file) but POSIX standard is `cp -R`, so should use that here and below.